libraries/plugins/account_history/account_history_plugin.cpp

```diff
@@ -206,13 +206,13 @@ void account_history_plugin::set_program_options(
    )
 {
    cfg.add_options()
-         ("account-history-track-account-range", boost::program_options::value< vector< string > >()->composing()->multitoken(), "Defines a range of accounts to track as a json pair [\"from\",\"to\"] [from,to] Can be specified multiple times.")
-         ("track-account-range", boost::program_options::value< vector< string > >()->composing()->multitoken(), "Defines a range of accounts to track as a json pair [\"from\",\"to\"] [from,to] Can be specified multiple times. Deprecated in favor of account-history-track-account-range.")
-         ("account-history-whitelist-ops", boost::program_options::value< vector< string > >()->composing(), "Defines a list of operations which will be explicitly logged.")
-         ("history-whitelist-ops", boost::program_options::value< vector< string > >()->composing(), "Defines a list of operations which will be explicitly logged. Deprecated in favor of account-history-whitelist-ops.")
-         ("account-history-blacklist-ops", boost::program_options::value< vector< string > >()->composing(), "Defines a list of operations which will be explicitly ignored.")
-         ("history-blacklist-ops", boost::program_options::value< vector< string > >()->composing(), "Defines a list of operations which will be explicitly ignored. Deprecated in favor of account-history-blacklist-ops.")
-         ("history-disable-pruning", boost::program_options::value< bool >()->default_value( false ), "Disables automatic account history trimming" )
+         ("ah-account-history-track-account-range", boost::program_options::value< vector< string > >()->composing()->multitoken(), "Defines a range of accounts to track as a json pair [\"from\",\"to\"] [from,to] Can be specified multiple times.")
```

Comment:
The pattern we are trying to use for all options is `plugin-name-option`. Prefixing this breaks the pattern because the plugin name is `account-history`, not `ah`. I also think that for the time being it is best to make the rocksdb options different so that the decision to use rocksdb is purposeful and not accidental.

Reply:
I have renamed such options (in legacy account-history plugin) because they conflicted to these defined in rocksdb plugin. Since we want to provide replacement for original AH plugin, I still think that is better to keep existing names of options for backward compatibility. That will make switch very easy at customer installations.